ui/components/app/signature-requests-common-header/signature-requests-common-header.js

```diff
@@ -0,0 +1,91 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { useSelector } from 'react-redux';
+import { useI18nContext } from '../../../hooks/useI18nContext';
+
+import {
+  getNativeCurrency,
+  getProviderConfig,
+} from '../../../ducks/metamask/metamask';
+import {
+  accountsWithSendEtherInfoSelector,
+  conversionRateSelector,
+  getCurrentChainId,
+  getCurrentCurrency,
+  getPreferences,
+} from '../../../selectors';
+import { formatCurrency } from '../../../helpers/utils/confirm-tx.util';
+import {
+  getAccountByAddress,
+  getNetworkNameFromProviderType,
+} from '../../../helpers/utils/util';
+import { EtherDenomination } from '../../../../shared/constants/common';
+import { Numeric } from '../../../../shared/modules/Numeric';
+import { getValueFromWeiHex } from '../../../../shared/modules/conversion.utils';
+
+import NetworkAccountBalanceHeader from '../network-account-balance-header';
+
+const SignatureRequestsCommonHeader = ({ txData }) => {
```

Comment:
In this PR, I have updated <SignatureRequestOriginal/> and <SignatureRequestSIWE/> to use this new component. This can also be used in <SignatureRequest/>, which I will be updating in https://github.com/MetaMask/metamask-extension/pull/19104  to reduce redundant work as I am refactoring the <SignatureRequest/> component to a functional component in that PR.